document_url/static/src/js/url.esm.js

```diff
@@ -29,6 +29,18 @@ registerPatch({
     },
 });
 
+registerPatch({
+    name: "Chatter",
+    recordMethods: {
+        /**
+         * Handles click on the attach button.
+         */
```

Comment:
nitpick:  maybe it'd be more informative to explain *why* we need to handle the click on the attach button.